azure-pipelines.yml

```diff
@@ -1,120 +1,120 @@
-              # was introduced which breaks use of securestring with ARM templates:
-              # https://github.com/Azure/azure-powershell/issues/12792
-              # https://github.com/Azure/azure-powershell/issues/12819
-              # Once this issue is fixed, we should revert back to using the latest version of the cmdlets.
-              preferredAzurePowerShellVersion: '4.4.0'
-              pwsh: true
-
-          - template: azurepipeline-templates/deploy-marain-instance.yml
-            parameters:
-              azureServiceConnection: $(Endjin_AzureServiceConnection)
-              azureSubscriptionId: $(Endjin_SubscriptionId)
-              azureTenantId: $(Endjin_AadTenantId)
-              azureLocation: $(Endjin_AzureLocation)
-              marainEnvironmentSuffix: $(Endjin_EnvironmentSuffix)
-              marainInstanceType: $(Marain_Instance_Type)
-              marainResourcePrefix: $(Marain_ResourcePrefix)
-              workingDirectory: $(Pipeline.Workspace)/marain_instance/Marain.Instance.Deployment
-
-- stage: Cleardown_Test_Deploynent
-  displayName: Cleardown test deployment
-  jobs:
-  - job: cleardown_test_deploynent
-    displayName: Cleardown test deployment
-    pool:
-      vmImage: 'ubuntu-latest'
-    steps:
-    - task: AzurePowerShell@4
-      condition: and(succeeded(), eq(variables['SkipCleardown'], ''))
-      displayName: 'Delete Azure resources'
-      inputs:
-        azureSubscription: '$(Endjin_AzureServiceConnection)'
-        ScriptType: InlineScript
-        Inline: |
-          $prefix = "$(Marain_ResourcePrefix)"
-          $suffix = "$(Endjin_EnvironmentSuffix)"
-          @("workflow","operations","tenancy","instance") | % {
-            $groupName = "{0}.{1}.{2}" -f $prefix, $_, $suffix
-            Write-Host ("Checking resource group: {0}" -f $groupName)
-            Get-AzResourceGroup $groupName -ErrorAction SilentlyContinue | Select -First 1 | Remove-AzResourceGroup -Verbose -Force
-          }
-          @("tenancy","operationscontrol","workfloweng","workflowmi","tenantadmin") | % {
-              $appName = "{0}{1}{2}" -f $prefix, $suffix, $_
-              Write-Host ("Checking AzureAD application: {0}" -f $appName)
-              Get-AzADApplication -DisplayName $appName | Select -First 1 | Remove-AzADApplication -Verbose -Force
-          }
-        # Temporarily force a downgrade of the AzurePowerShell cmdlets because (as of September 2020 and v4.6.0) a bug
-        # was introduced which breaks use of securestring with ARM templates:
-        # https://github.com/Azure/azure-powershell/issues/12792
-        # https://github.com/Azure/azure-powershell/issues/12819
-        # Once this issue is fixed, we should revert back to using the latest version of the cmdlets.
-        preferredAzurePowerShellVersion: '4.4.0'
-        pwsh: true
+# Starter pipeline

```

Comment:
Not sure why this file is showing changes. Is it line endings? Maybe we should have a .gitattributes file to make line endings consistent across Windows/*nix

Reply:
Line endings renormalized